collect_app/src/androidTest/java/org/odk/collect/android/test/FormLoadingUtils.java

```diff
@@ -52,6 +52,7 @@ private FormLoadingUtils() {
      */
     public static void copyFormToStorage(String formFilename, List<String> mediaFilePaths, boolean copyToDatabase) throws IOException {
         new StorageInitializer().createOdkDirsOnStorage();
+        ReferenceManager.instance().reset();
```

Comment:
Shouldn't it be in ResetStateRule?

Reply:
It was originally. I actually ended up on a call with @lognaturel to double check all the JavaRosa stuff (as I'm not as knowledgable about all that) and we ended up deciding to put this here. The reasoning was that the app doesn't need `reset()` to be called (it already handles setting up the `ReferenceManager`) but `CopyFormRule` does need it.

Reply:
ok